Fix alignment issue in AdditionalFees component

lisunshiny opened the pull request:
Fix bloom-housing/bloom#3220 by removing some unnecessary margin classes (maybe cruft from a previous world where this component had application fee & deposit side by side?).

I haven't used GitHub properly in years, plz advise if I miss anything!

lisunshiny commented:
@slowbot gotcha. Question -- would you recommend still using the InfoCard component here, or would you favor Card (https://storybook.bloom.exygy.dev/?path=/docs/blocks-card--text-content), or a different component altogether? Card has some nice stuff like a footer, which would be useful here.

I figure if the job is to match the AdditionalFees to the spec the best way is to use the component library directly instead of these divs/classes, but am not sure which one. WDYT?

lisunshiny commented:
@slowbot  -- actually I decided that bringing the whole component up to spec is outside of the scope of this change, but I did make some changes to bring the two columns back. imo this is a strict improvement. lmk!

emilyjablonski commented on src/page_components/listing/AdditionalFees.tsx:
```diff
@@ -1,4 +1,9 @@
 import * as React from "react"
+import {
```
Let's import these from their individual files like `import { GridSection } from "../../sections/GridSection"`

lisunshiny replied:
done

emilyjablonski reviewed (approved):
Great q about which component to pull in - we definitely prefer an existing component in use here so this is a great update! Agreed that getting us fully up to spec is a separate issue and this is a good improvement. After one more approval & an update to the imports this is ready to merge :)

lisunshiny commented:
Thanks for the review @emilyjablonski! Updated with your change and also fixed the double margin UI annoyance in a better way

Do you know who I should tag for review number 2 (and more generally, who I should tag to review CLs to bloom repos?)

Lastly I squash commit right? If I also missed any of the CL review best practices lmk.

ColinBuyck reviewed (approved):
This all LGTM 🌲 Thank you for making these improvements! In terms of merging, squash commit is best. I'll have to defer to @emilyjablonski for the CL best practices portion of your question though.

emilyjablonski commented:
Since this has two approvals and I'm not sure we've set up git permissions yet, I'll go ahead and merge this one - thank you!

lisunshiny commented:
SG! Feel free to merge whenever, you're right I don't have access 😓. Lots of useful info in https://github.com/bloom-housing/ui-components/pull/8#issuecomment-1396300315 so I shared it with the team as well so they know, and sg. re slack channel. Thanks for the speedy reviews
